src/main/code.ts

```diff
@@ -177,15 +183,17 @@ function createTable(
 
 function getFirstChildOfTypeText(sceneNode: SceneNode) {
 	if ("children" in sceneNode) {
-		return sceneNode.findOne(node => node.type === "TEXT") as TextNode
-	} else {
-		return null
+
+		let textNodes = sceneNode.findAll(node => node.type === "TEXT")
+		return textNodes[0] as TextNode
 	}
```

Comment:
* Question: Is it not necessary to add `!node.visible` to the query?
* Suggestion: Put `figma.skipInvisibleInstanceChildren = true` right in front of line 187? That way it would be easier to understand what the intention of that line is
* Improvment: Instead of use `findAll` and subsequently return the first element of its result, you'd better use `findOne` which stops searching after the first hit. Actually, just use the code as it was